docs/UserGuide.md

```diff
@@ -182,15 +267,15 @@ the data of your previous AcademyAssist folder.
 
 ## Command summary
 
-| Action | Format, Examples |
-|--------|-------------------|
-| **Add** | `add n/NAME ic/IC_NUMBER e/EMAIL p/PHONE_NUMBER a/ADDRESS c/CLASS y/ACADEMIC_YEAR` <br> e.g., `add n/John Doe ic/T384859A e/johndoe@gmail.com p/81003999 a/9 Smith Street c/Science1 y/Standard1` |
-| **Delete** | `del STUDENT_ID`<br> e.g., `del 12345` |
-| **Edit** | `edit STUDENT_ID FIELD:NEW_VALUE`<br> e.g.,`edit 12345 Address:New Address` |
-| **View** | `view` |
-| **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find John Jane` |
-| **Add Class** | `addc STUDENT_ID CLASS_NAME`<br> e.g., `addc 12345 Science1` |
-| **Sort** | `sort FIELD`<br> e.g., `sort name` |
-| **Clear** | `clear` |
-| **Help** | `help` |
-| **Exit** | `exit` |
+| Action | Format, Examples                                                                                                                                                           |
+|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS i/IC_NUMBER s/SUBJECT t/TAG` <br> e.g., `add n/John Doe p/81003999 e/johndoe@gmail.com a/9 Smith Street i/T384859A s/Science` |
+| **Delete** | `delete IC_NUMBER`<br> e.g., `delete S1234567A`                                                                                                                            |
+| **Edit** | `edit IC_NUMBER FIELD/NEW_VALUE`<br> e.g.,`edit S1234567A a/New_Address`                                                                                                   |
+| **View** | `view`                                                                                                                                                                     |
+| **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find John Jane`                                                                                                                  |
```

Comment:
Since we are only finding based on student name, perhaps we can just put find NAME instead of KEYWORD as a user may misunderstand a KEYWORD as subject name for example. And since name can be understood as just first name, full name, last name etc maybe the [MORE_KEYWORDS] at the back won't be needed as well